run

```diff
@@ -515,6 +515,9 @@ run_mkdrive() {
             return
         fi
     fi
+    if ! lsmod | grep loop; then
+        modprobe loop
```

Comment:
The fix won't work on macOS as it doesn't have lsmod and friends. As of now your fix only works if you run ktest on Linux.

Reply:
losetup error occurs only on Linux.
I add checking OS before using lsmod and modprobe.

Reply:
> The fix won't work on macOS as it doesn't have lsmod and friends. As of now your fix only works if you run ktest on Linux.

Now we start docker container and load loop module on Mac OS before running mkfs.

Reply:
It turned out macOS has loop built-in, but the docker invocation is useful as it doesn't require sudo.